pdc/apps/auth/views.py

```diff
@@ -141,16 +141,22 @@ def user_profile(request):
 
 
 def get_users_and_groups(resource_permission):
+    group_resource_permission_list = []
     try:
         group_resource_permission_list = get_list_or_404(models.GroupResourcePermission,
                                                          resource_permission=resource_permission)
     except Http404:
```

Comment:
I think here is condition `group_resource_permission_list` being false.
You could use
```
try:
.....
except Http404:
....
else:
...........

```